amy/workshops/tests/test_tasks.py

```diff
@@ -369,6 +371,48 @@ def test_seats_for_learners_only(self):
         task3.full_clean()
         task4.full_clean()
 
+    def test_seat_public_validation(self):
```

Comment:
I think it might be a bit better to separate this out into two tests as it stands if you have an issue where you're not checking the right value in the validation you wont catch it. I.e. if task2 for instance was throwing a validation because of public_instructor_training_seats. 

It's obviously not a bug you're encountering now but I think it makes for a stronger test.

Reply:
Resolved.